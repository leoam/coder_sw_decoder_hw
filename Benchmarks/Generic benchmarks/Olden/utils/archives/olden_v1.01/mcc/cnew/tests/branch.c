
typedef struct this{
	int x;
} *This;

extern void *mymalloc();

main()
{
	int y;
	This thing,true;
	true = mymalloc(sizeof(struct this));
        MIGRATE();
        MIGRPH();
	thing = mymalloc(sizeof(struct this));
	if(y)
	{thing->x = 2;}
	else true->x = 7;
	y = thing->x;
	y = true->x;
	y = thing->x;
}	
