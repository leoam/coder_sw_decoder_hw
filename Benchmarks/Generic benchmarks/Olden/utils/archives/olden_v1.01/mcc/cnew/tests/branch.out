10: operands of = have illegal types `pointer to struct this' and `int'
13: operands of = have illegal types `pointer to struct this' and `int'
.seg "text"

.global _main
-----------------------
************************
-----------------------
************************
-----------------------
true 
local:true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77520
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77570
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:775d0
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77620
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77680
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:776d0
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77730
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77780
all_work: 0
Nsucc: 2  howfar: 0
i:0 list:77880
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:778e0
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77930
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:779a0
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:779f0
all_work: 0
Nsucc: 0  howfar: 0
